plugin/download.sh

```diff
@@ -64,6 +66,10 @@ download_assets_for_release() {
     # Node 19
     download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-19-$tag.sha512" "$dir/heroku-nodejs-plugin-node-19.sha512"
     download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-19-$tag.tar.gz" "$dir/heroku-nodejs-plugin-node-19.tar.gz"
+
+    # Node 20
+    download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-19-$tag.sha512" "$dir/heroku-nodejs-plugin-node-20.sha512"
```

Comment:
Is it correct that we download the node 19 version for node 20? Are they compatible and don't warrant a new plugin?

Reply:
no, i missed updating a 19 to 20 here.

Reply:
fixed in latest commit